package.json

```diff
@@ -19,9 +19,13 @@
     "compression": "^1.5.2",
     "errorhandler": "^1.4.3",
     "express": "^4.13.3",
+    "express-minify": "^0.2.0",
     "express-sslify": "^1.0.1",
-    "jstransformer-uglify-js": "^1.1.1",
+    "fs-walk": "0.0.1",
+    "html-validator": "^1.0.0",
+    "htmlencode": "0.0.4",
     "js-yaml": "^3.4.2",
+    "jstransformer-uglify-js": "^1.1.1",
```

Comment:
Actually it was already there just higher up, but probably not. As I said, I'll verify.